src/components/Questionnaire/ValueSetSelect.tsx

```diff
@@ -58,6 +71,9 @@ export default function ValueSetSelect({
   const [internalOpen, setInternalOpen] = useState(false);
   const [search, setSearch] = useState("");
   const isMobile = useBreakpoints({ default: true, sm: false });
+  const [activeTab, setActiveTab] = useState(0);
+  const [isClearingFavourites, setIsClearingFavourites] = useState(false);
```

Comment:
this state is unnecessary right? `clearFavouritesMutation.isPending` is the equivalent of this right? 🤔 

```suggestion
```

Reply:
Its not for loading, its for showing the clear favourites prompt